### DISCUSSION TITLE
为什么我不挂梯子也能访问？

### DISCUSSION THREAD

**Discussion by Ziew-Ryan** [share121/inter-knot#2733] [其他]
本人IP云南，试了一下竟然能不挂梯子直连绳网和Github，好神奇，是Firewall放过了我吗？

**Comment by miaochu1** [comment]
其实Github可以直连的只是非常不稳定，也许你运气好<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/dec64f96a41d05d0f0c73432e7636e7b_5424104392001050566.png" alt="米游君——认真工作" width="75" >

**Reply by Ziew-Ryan** [reply]
确实不稳定，5分钟前能连上，3分钟前网站无响应，现在又能访问了

**Reply by cyq20091029** [reply]
鄙人也是云南，githup有时候的确可以直连，但很看运气

**Comment by Pleasurecruise** [comment]
可以试试CSDN的github加速计划......

**Reply by Ziew-Ryan** [reply]
现在是用着Steam++自带的加速服务
